org.godotengine.Godot3.yaml

```
@@ -0,0 +1,94 @@
+app-id: org.godotengine.Godot3
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: godot
+
+build-options:
+  arch:
+    x86_64:
+      env:
+        # Only enable link-time optimization when targeting x86_64
+        # (causes issues on other architectures)
+        SCONS_FLAGS_EXTRA: use_lto=yes
```

Use up-to-date SCons option for enabling LTO:

```suggestion
        SCONS_FLAGS_EXTRA: lto=full
```